ISSUE_TEMPLATE/membership.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Organization Membership Request
+about: Request membership in a OpenFeature Org
```

Comment:
This doesn't seem grammatically correct perhaps "Request membership in the OpenFeature Org